foil.tex

```diff
@@ -72,11 +65,11 @@ \chapter{Four-digit NACA Airfoil}
     \item
     Camber line (red dashed line)
     \item
-    Leading edge ($x = 0, y = 0$)
+    Leading edge ($x=0, y=0$)
```

Comment:
Please revert unnecessary change of blank lines and other minor formatting.

In a PR adding and changing (editing) contents, formatting changes should be avoided.  On the other hand, when changing formatting/typesetting, do not make content change.  Mixing content and formatting changes will make review hard.

This is a principle derived from computer code.